packages/modular-scripts/src/build/index.ts

```diff
@@ -258,21 +238,21 @@ async function buildStandalone(
   const targetPackageJson = (await fs.readJSON(
     path.join(targetDirectory, 'package.json'),
   )) as CoreProperties;
-  targetPackageJson.dependencies = packageDependencies;
-  targetPackageJson.bundledDependencies = Object.keys(bundledDependencies);
-
   // Copy selected fields of package.json over
-  await fs.writeJSON(
+  fs.writeJSON(
     path.join(paths.appBuild, 'package.json'),
     {
       name: targetPackageJson.name,
       version: targetPackageJson.version,
       license: targetPackageJson.license,
       modular: targetPackageJson.modular,
-      dependencies: targetPackageJson.dependencies,
-      bundledDependencies: targetPackageJson.bundledDependencies,
+      dependencies: packageDependencies,
+      bundledDependencies: isApp
+        ? Object.keys(packageDependencies)
+        : Object.keys(bundledResolutions),
       module: jsEntryPoint ? paths.publicUrlOrPath + jsEntryPoint : undefined,
       style: cssEntryPoint ? paths.publicUrlOrPath + cssEntryPoint : undefined,
+      'style-cdn': styleImports?.size ? [...styleImports] : undefined,
```

Comment:
Could this follow the naming convention of `camelCase` rather than `kebab-case` for the property?

As these are `styleImports` elsewhere, might that be a useful name here?

Reply:
Set it to `styleImports`, it seems that nobody else's using it, so we can claim it in this npm Wild West :D